src/quantuminspire/projectq/backend_qx.py

```diff
@@ -60,13 +58,15 @@ def __init__(self, num_runs: int = 1024, verbose: int = 0, quantum_inspire_api:
         """ Because engines are meant to be 'single use' by the way ProjectQ is designed,
         any additional gates received after a FlushGate triggers an exception. """
         self._clear: bool = True
+        self.qasm: str = ""
```

Comment:
Is `qasm` part of the public interface of QIBackend? If not, it should be prefixed with an underscore.

Reply:
Done